auslib/admin/views/releases.py

```diff
@@ -507,3 +507,22 @@ def _post(self, changed_by, transaction):
         )
         response.headers['Content-Type'] = 'application/json'
         return response
+
+
+class SingleReleaseColumnView(AdminView):
+    """ /releases/column/:column"""
+
+    def get(self, column):
+        releases = dbo.releases.getReleases()
+        column_values = []
+
+        for release in releases:
+            for key, value in release.items():
+                if key == column and value is not None:
+                    column_values.append(value)
+        column_values = list(set(column_values))
+        ret = {
+            "count": len(column_values),
+            column + "s": column_values,
```

Comment:
No need to pluralize here. Adding an "s" won't work for some columns anyways (eg: Rules.alias becomes aliass).